wedlocks/src/templates/keyMined.js

```diff
@@ -9,6 +9,8 @@ export default {
 
 <p>A new membership (#{{keyId}}) to the lock <strong>{{lockName}}</strong> was just minted for you!</p>
 
+{{customEmailContent}}
```

Comment:
@julien51 this will cause extra spacing when the custom content is not present? 🤔

Reply:
Use conditional 

https://handlebarsjs.com/examples/builtin-helper-ifelse-block.html